codice/ddf
# DDF-2025 - updated ActionProvider to return a List of Actions. Added …

#### What does this PR do?
Updates the experimental ActionProvider interface to return a List<Action>.
Adds DerivedContentActionProvider as an implemenation.
#### Who is reviewing it (please choose AT LEAST two reviewers that need to approve the PR before it can get merged; if a component team is listed, at least one of its members needs to approve)?
@jaymcnallie @dcruver 
#### Choose 2 committers to review/merge the PR (please choose ONLY two committers from below, delete the rest).
@kcwire
@pklinef
#### How should this be tested?
Verify Integration tests. For a live Install the sample-storageplugins bundle, ingest a jpg through the search ui, verify the derived content uris are in the details tab & the new action under the Actions tab downloads a thumbnails of the jpg.
#### Any background context you want to provide?
#### What are the relevant tickets?
DDF-2025
#### Screenshots (if appropriate)
#### Checklist:

…DerivedContentActionProvider for derived content items.
